app/pb_kits/playbook/pb_nav/_nav.scss

```diff
@@ -0,0 +1,143 @@
+@import "../tokens/typography";
+@import "../tokens/spacing";
+@import "../tokens/colors"; // Another style should be borders like buttons
+$selector: ".pb_nav_list";
+
+[class^=pb_nav_list][class*=_vertical] {
```

Comment:
I think it would be great to break horizontal and vertical into their own sass files and import them here